Detection of Small Flying Objects in UAV Videos/README.md

```diff
@@ -0,0 +1,92 @@
+# Detection of Small Flying Objects in UAV Videos
```

Comment:
Missing a link to the DagsHub repo

Reply:
The link to the DagsHub repo is in line 11 of the readme, but I can shift it up later if you'd like. Re: the other changes, will do tomorrow!